src/openproduct/producten/tasks.py

```diff
@@ -5,8 +5,9 @@
 
 
 def updated_based_on_dates():
-    for product in Product.objects.all():
-        product.save()
+    for product in Product.objects.iterator():
+        if product.check_start_datum() or product.check_eind_datum():
+            product.save()
```

Comment:
This does result in two queries being done for every item for which either `check_start_datum()` or `check_eind_datum` is `True`. I don't know how likely it is for this queryset to become huge, but if that's a possibility, it might be good to check if it's possible to do things in bulk here.

Reply:
created #92 for performance stuff for after the mvp